src/core/pages/AccountInfo.vue

```diff
@@ -117,11 +117,10 @@ export default {
       },
     }
   },
-  async mounted () {
+  async created () {
     try {
-      this.mergedUserProfile.fromBack = await Users.getById(this.$route.params.id);
-      this.mergedUserProfile.local.email = this.mergedUserProfile.fromBack.local.email;
-      this.mergedUserProfile.contact.phone = get(this.mergedUserProfile.fromBack, 'contact.phone') || '';
+      const user = await Users.getById(this.$route.params.id);
+      this.mergedUserProfile = pick(user, ['local', 'contact']);
```

Comment:
Ici il faut en fait que tu prennes que 'local.email' sinon dans le champs nvx mdp de la modale tu a l'ancien mdp.

Reply:
et il faut que tu mettes `this.mergedUserProfile = { contact: {}, ...pick(user, ['local.email', 'contact']) };` pour que `contact` soit defini dans tous les cas

Reply:
done, dsl je suis allé vite la dessus